src/store/messages/saga.ts

```diff
@@ -253,28 +253,32 @@ export function* createOptimisticPreview(channelId: string, optimisticMessage) {
 }
 
 export function* performSend(channelId, message, mentionedUserIds, parentMessage, optimisticId) {
-  const chatClient = yield call(chat.get);
+  try {
+    const chatClient = yield call(chat.get);
+    const messageCall = call(
+      [chatClient, chatClient.sendMessagesByChannelId],
+      channelId,
+      message,
+      mentionedUserIds,
+      parentMessage,
+      null,
+      optimisticId
+    );
+    const createdMessage = yield messageCall;
 
-  const messageCall = call(
-    [
-      chatClient,
-      chatClient.sendMessagesByChannelId,
-    ],
-    channelId,
-    message,
-    mentionedUserIds,
-    parentMessage,
-    null,
-    optimisticId
-  );
-  return yield sendMessage(messageCall, channelId, optimisticId);
```

Comment:
So we no longer want to `sendMessage`? This feels incorrect to remove

Reply:
This appears to be working because you are no longer replacing the optimistic message

Reply:
Thanks @domw30, apparently everything is working (Reply and editing messages).
I will return with sendMessage, preventing unnecessary re-rendering.

Reply:
https://github.com/zer0-os/zOS/assets/57037080/ecc44d19-39a3-405f-a6d2-403fb0c47fc3